quill-core/src/test/scala/io/getquill/ast/AstShowSpec.scala

```diff
@@ -505,4 +505,35 @@ class AstShowSpec extends Spec {
     (q.ast: Ast).show mustEqual
       """query[TestEntity].distinct"""
   }
+
+  "shows schema" - {
+    "table" - {
+      val q = quote {
+        query[TestEntity].schema(s => s.table("test"))
```

Comment:
entity sounds better to me
On 1 Mar 2016 00:33, "Gustavo Amigo" <notifications@github.com> wrote:

> In quill-core/src/test/scala/io/getquill/ast/AstShowSpec.scala
> <https://github.com/getquill/quill/pull/226#discussion_r54505250>:
>
> > @@ -505,4 +505,35 @@ class AstShowSpec extends Spec {
> >      (q.ast: Ast).show mustEqual
> >        """query[TestEntity].distinct"""
> >    }
> > +
> > +  "shows schema" - {
> > +    "table" - {
> > +      val q = quote {
> > +        query[TestEntity].schema(s => s.table("test"))
>
> @fwbrasil <https://github.com/fwbrasil> what do you think if we use entity
> instead of table?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/getquill/quill/pull/226/files#r54505250>.
>